circular-progressbar/src/main/scala/react/circularprogressbar/CircularProgressbar.scala

```diff
@@ -93,5 +93,5 @@ object CircularProgressbar {
     p
   }
 
-  private val component = JsComponent[Props, Children.None, Null](RawComponent)
+  private val component = JsComponent.force[Props, Children.None, Null](RawComponent)
```

Comment:
Why this?

Reply:
Without it, I get this. So a scalajs-react bug?

```
================================================================================
Invalid JsComponent at
/workspace/lucuma-react/circular-progressbar/src/main/scala/react/circularprogressbar/CircularProgressbar.scala:96

You called JsComponent() with an invalid argument:
  function CircularProgressbar() {
          return _super !== null && _super.apply(this, arguments) || this;
      }: function

Make sure that
  * your @JSImport / @JSGlobal annotations have the correct values
  * the JS that you're referencing has been loaded into the JS environment

If you believe this error message is wrong, please raise a scalajs-react issue
and use JsComponent.force as a workaround.
```

Reply:
It's basically the same, just without validation.

https://github.com/japgolly/scalajs-react/blob/ba8805008ae7a88f61e616a91ee9815e5795f8e3/library/coreGeneric/src/main/scala/japgolly/scalajs/react/component/Js.scala#L14-L24